salt/modules/supervisord.py

```diff
@@ -245,10 +245,13 @@ def reread(user=None, conf_file=None, bin_env=None):
     return _get_return(ret)
 
 
-def update(user=None, conf_file=None, bin_env=None):
+def update(name=None, user=None, conf_file=None, bin_env=None):
```

Comment:
There may be people calling this positionally. It would be better to move the new kwarg to the end of the list.